src/notehead.ts

```diff
@@ -38,66 +29,11 @@ export interface NoteHeadStruct extends NoteStruct {
   index?: number;
 }
 
-/**
- * Draw slashnote head manually. No glyph exists for this.
- * @param ctx the Canvas context
- * @param duration the duration of the note. ex: "4"
- * @param x the x coordinate to draw at
- * @param y the y coordinate to draw at
- * @param stemDirection the direction of the stem
- */
-function drawSlashNoteHead(
-  ctx: RenderContext,
-  duration: string,
-  x: number,
-  y: number,
-  stemDirection: number,
-  staveSpace: number
-) {
-  const width = Tables.SLASH_NOTEHEAD_WIDTH;
-  ctx.save();
-  ctx.setLineWidth(Tables.STEM_WIDTH);
-
-  let fill = false;
-
-  if (Tables.durationToNumber(duration) > 2) {
-    fill = true;
-  }
-
-  if (!fill) x -= (Tables.STEM_WIDTH / 2) * stemDirection;
-
-  ctx.beginPath();
-  ctx.moveTo(x, y + staveSpace);
-  ctx.lineTo(x, y + 1);
-  ctx.lineTo(x + width, y - staveSpace);
-  ctx.lineTo(x + width, y);
-  ctx.lineTo(x, y + staveSpace);
-  ctx.closePath();
-
-  if (fill) {
-    ctx.fill();
-  } else {
-    ctx.stroke();
-  }
-
-  if (Tables.durationToFraction(duration).equals(0.5)) {
-    const breveLines = [-3, -1, width + 1, width + 3];
-    for (let i = 0; i < breveLines.length; i++) {
-      ctx.beginPath();
-      ctx.moveTo(x + breveLines[i], y - 10);
-      ctx.lineTo(x + breveLines[i], y + 11);
-      ctx.stroke();
-    }
-  }
-
-  ctx.restore();
-}
-
 /**
  * `NoteHeads` are typically not manipulated
  * directly, but used internally in `StaveNote`.
  *
- * See `tests/notehead_tests.ts` for usage examples.
+ * See `tests/noteheadTests.ts` for usage examples.
```

Comment:
This is a typo (due to camelCase conversion).

Reply:
done